[2025-05-08T15:18:15.135+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-08T15:18:15.186+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: e_commerce_dw_etl_decorator.transform.transform_dim_customers manual__2025-05-08T15:17:25.111912+00:00 [queued]>
[2025-05-08T15:18:15.203+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: e_commerce_dw_etl_decorator.transform.transform_dim_customers manual__2025-05-08T15:17:25.111912+00:00 [queued]>
[2025-05-08T15:18:15.204+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-05-08T15:18:15.227+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform.transform_dim_customers> on 2025-05-08 15:17:25.111912+00:00
[2025-05-08T15:18:15.239+0000] {standard_task_runner.py:63} INFO - Started process 12193 to run task
[2025-05-08T15:18:15.245+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'e_commerce_dw_etl_decorator', 'transform.transform_dim_customers', 'manual__2025-05-08T15:17:25.111912+00:00', '--job-id', '148', '--raw', '--subdir', 'DAGS_FOLDER/ecommerce_dw_dag.py', '--cfg-path', '/tmp/tmput4z22g9']
[2025-05-08T15:18:15.250+0000] {standard_task_runner.py:91} INFO - Job 148: Subtask transform.transform_dim_customers
[2025-05-08T15:18:15.334+0000] {task_command.py:426} INFO - Running <TaskInstance: e_commerce_dw_etl_decorator.transform.transform_dim_customers manual__2025-05-08T15:17:25.111912+00:00 [running]> on host ebff8b55cad8
[2025-05-08T15:18:15.487+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='e_commerce_dw_etl_decorator' AIRFLOW_CTX_TASK_ID='transform.transform_dim_customers' AIRFLOW_CTX_EXECUTION_DATE='2025-05-08T15:17:25.111912+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-08T15:17:25.111912+00:00'
[2025-05-08T15:18:15.489+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-08T15:18:15.502+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2025-05-08T15:18:15.521+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/providers/common/sql/hooks/sql.py:263 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-05-08T15:18:16.347+0000] {logging_mixin.py:188} INFO - Transformed and saved data to dim_customers
[2025-05-08T15:18:16.363+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-05-08T15:18:16.364+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-08T15:18:16.377+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=e_commerce_dw_etl_decorator, task_id=transform.transform_dim_customers, run_id=manual__2025-05-08T15:17:25.111912+00:00, execution_date=20250508T151725, start_date=20250508T151815, end_date=20250508T151816
[2025-05-08T15:18:16.446+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-08T15:18:16.489+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-08T15:18:16.493+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
